index.html

```diff
@@ -1681,45 +1682,26 @@ <h2>The <dfn>record type</dfn> string</h2>
         <dd>
           The value representing an <a>unknown record</a>.
         </dd>
-        <dt>An <dfn>external type name</dfn></dt>
-        <dd>
-          A {{DOMString}} representing a custom type for the
-          <a>external type record</a>.
-          The type must follow the [=external type name=] ABNF.
-          <pre class="abnf">
-            ext-type             = reg-name ":" custom-type
-            custom-type          = 1*(ALPHA / DIGIT / other)
-
-            DIGIT                = %x30-39
-            ALPHA                = %x41-5A / %x61-7A   ; A-Z / a-z
-            other                = "(" / ")" / "+" / "," / "-" / ":" / "=" /
-                                   "@" / ";" / "$" / "_" / "!" / "*" / "'" / "."
-          </pre>
-          The `reg-name` value is a [=host/registrable domain=] owned by the issuing
-          organization, a "`:`" and a type, e.g. "`w3.org:member`".
-          And additional ABNF exists for [=well-known type records=]:
-          <pre class="abnf">
-            wkt-type             = (ALPHA / DIGIT) *(ALPHA / DIGIT / other)
-          </pre>
-          <p class=note>
-            The [[[NFC-RTD]]] defines every type in the [=well-known type records=]
-            and <a>external type records</a> in terms of URNs, but only a subset of
-            the URN is actually stored in the <a>NDEF record</a>'s <a>TYPE field</a>,
-            which corresponds to the above two ABNFs.
-          </p>
-        </dd>
-        <dt>A <dfn>local type name</dfn></dt>
-        <dd>
-          A {{DOMString}} that MUST start with lowercase character or a number,
-          representing a type for a NFC Forum [=local type=],
-          typically used in a record of an <a>NDEFMessage</a> that is the payload
-          of a parent <a>NDEFRecord</a>, for instance in a <a>smart poster</a>.
-          The context of the <a>local type</a> is the parent record whose payload
-          is the <a>NDEFMessage</a> to which this record belongs. The value
-          MUST NOT be equal to any other <a>record type</a>s defined
-          in this API.
-        </dd>
       </dl>
+      <p>
+        In addition to [=well known type names=] it is also possible for
+        organizations to create a own custom <dfn>external type name</dfn> that
+        which is a string consisting of a domain name and a custom ASCII name,
+        separated by a colon (`':'`). The string may not contain any whitespace
```

Comment:
I think my other comment is probably better

Reply:
This is not the place we specify the algorithm, and we should avoid repeating that complex thing. So either we say "satisfies the validate external type name steps", or we give a short summary in words.

Reply:
In addition to [=well known type names=] it is also possible for organizations to create a own custom <dfn>external type name</dfn> which is a string consisting of a domain name and a developer created type.

The developer created type code points are ASCII alphanumeric, U+0024 ($), U+0027 ('), U+0028 LEFT PARENTHESIS, U+0029 RIGHT PARENTHESIS, U+002A (*), U+002B (+), U+002C (,), U+002D (-), U+002E (.), U+003B (;), U+003D (=), U+0040 (@), U+005F (_).

Reply:
That seems fine and aligned with other specs.

The NFC spec made it complex

Reply:
Can we keep that enumeration in the validate external type steps only, and refer to them from here?

Reply:
URL specs uses U+00** when describing and 0x** when used in algos. I thikn this is fine. It is a spec, not documentation :)

Reply:
OK, I took another attempt on it (amend).

Reply:
Removed the validation specific part.